dts/bindings/pwm/nxp,s32-emios-pwm-capture.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  NXP S32 eMIOS PWM pulse capture node for S32 SoCs
+
+compatible: "nxp,s32-emios-pwm-capture"
```

Comment:
My concern with this approach is that we are introducing two bindings for the same hw block, but each binding is addressing a different functionality, generate and capture, of PWM. I think it would be better to have a single binding and device driver for the EMIOS PWM that is used for both capture and generation. Unless there's something particular to this device (hw) that doesn't allows us to do that.

Reply:
ok, i've updated to use single shim driver for both